apps/native-component-list/webpack/webpack.common.js

```diff
@@ -305,21 +297,20 @@ module.exports = {
 
     new webpack.DefinePlugin(env),
 
-    useWebModule('Platform', 'Utilities/Platform'),
     useWebModule('Performance/Systrace', 'Performance/Systrace'),
-    useWebModule('HMRLoadingView', 'Utilities/HMRLoadingView'),
     useWebModule('RCTNetworking', 'Network/RCTNetworking'),
-    
+    /** Upstreamed */
```

Comment:
What does this mean?

Reply:
It's a part of RN and eventually we'd remove it from here. But I just deleted the comment cuz there's a chance I could remove it before then.